charts/common/values.yaml

```diff
@@ -1,58 +1,125 @@
-# app: the-app
-# shortname: theapp
-# team: yourteam
-# env: dev
+# -- Application name, typically on the form `the-application`
+app:
+# -- `id` for GCP 2.0, typically on the form `theapp`
+shortname:
+# -- Your team name, without a `team-` prefix
+team:
+# -- The current env, override in your `values-kub-ent-$env.yaml` files to `dev`, `tst` or `prd`
+env:
```

Comment:
Consider adding empty strings instead of `null` to make the variable type more visible. Or comment out the line with a example value.

Reply:
Commenting out the stanza will lead to it not being documented. Setting a default value such as `""` will lead to much uglier checks as we now can simply check for a value, and not string compare against an empty or check the length.
`null` in this case is a `required`  indicator.